### PR TITLE
Add ability to merge pull requests

### PR DESCRIPTION
**Pull request by ghost**
Policy code now goes in `pr-info.ts` and `use-pr-info.ts`. `pr-info` gathers info, `use-pr-info` has utility functions for getting concrete actions (label names, comment text). `rules.ts` actually executes actions. Other modules are just for gathering information and shouldn't have policy code.

Based on discussion in DefinitelyTyped/DefinitelyTyped#18449, we will automatically merge if:
* At least one owner approves
* There are no negative reviews
* The PR touches only one package, and doesn't touch any files not in packages
* It's not a popular package (>200,000 downloads monthly)
* The PR has been around for at least 2 days.

### CODE REVIEW

File: src/reviews.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+function getProperReviews(
+    reviewsFromApi: ReadonlyArray<bot.PullRequestReview>,
+    prUser: string,
+    reviews: Push<Review>,
+    mergeRequesters: Push<string>): void {
+    for (const r of reviewsFromApi) {
+        const reviewer = r.user.login;
+        // No self-reviews
+        if (reviewer === prUser)
```
Comment (RyanCavanaugh) [inline src/reviews.ts:104]: What happened to the check for `dt-bot`'s comments?

File: src/rules.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        // Skip issues and closed PRs
+        if (!(pr.isPullRequest && pr.state === "open"))
+            return;
+
+        const info = await getPRInfo(pr);
+        console.log(pr.fullName, info);
+
+        // Move to appropriate project
+        const column = getProjectColumn(info);
+        if (column !== undefined)
```
Comment (RyanCavanaugh) [inline src/rules.ts]: This should be made able to remove PRs from all columns if applicable

File: src/reviews.ts (hunk 1 above)
Comment (ghost) [reply src/reviews.ts:104]: Added back.

**Comment by RyanCavanaugh** [comment]
Will merge this once the current catch-up run finishes

**Pull request merged**